README.md

```diff
@@ -1,4 +1,108 @@
+[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/openfeatureflags/.github/blob/main/CONTRIBUTING.md)
+
+This document describes the bootstrap governance process under which the project will operate
+until the final governance process is identified.
+
+> :warning: This is a temporary (aka bootstrap) governance document that
+> is effective until the project is fully established.
+> A final process 
+
+## Project manifesto
+
+### Our goal
+
+Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+
+### Scope
+
+The scope includes but not limited to:
+
+- An **open standard** that allows adopters and vendors to easily integrate with spec-compliant feature flag solutions
+- Creating a **vendor-agnostic ecosystem** supported by the CNCF that provide a seamless experience for developers, SREs, and vendors
+- Providing Unified API and SDK;
+- Developer-first, cloud-native reference implementation; 
+- Ensuring extensibility for open source and commercial offerings.
+
+### Why OpenFeature?
+
+There are many existing solutions, open source or not.
+Unfortunately they lack key features needed for wide adoption
+by companies and in the open source ecosystem:
+
+- **Many SDKs** doing the same thing, but slightly differently
+- Many companies **“rebuilding” the wheel**
+- SDKs are generally open source but have no ecosystem
+- **“Vendor” lock-in** at code-level
+- **No collaboration** across vendors. 
+- **Missing integration** with observability tools (e.g. OpenTelemetry)
+
+### Project Differentiators
+
+This project is more than just copy and paste of existing solutions
+
+* Current solutions often have **eval logic in SDKs**.
+  This may require **updating all your apps** for a new rule type.
+* Config in Feature Flagging tools that are **separate from the rest of the app** make it harder to use approaches like GitOps. 
+* Config and rule changes are **hard to integrate into observability** tools 
+* Information to decide on features already available in observability tools is **not leveraged and collected twice**
+* No **agreed on grammar/structure** for rules which could be automatically validated (e.g. with CUE)
+
+## Roadmap
+
+The project roadmap is available [here](https://github.com/orgs/openfeatureflags/projects/1).
+New initiatives for the roadmap can be suggested via marking an issue as `roadmap-proposal` and then reaching a community decision as documented in _Decision Making_.
+
+- [Initiatives on the project roadmap](https://github.com/search?q=org%3Aopenfeatureflags+label%3Aroadmap&type=issues)
+- [Pending roadmap suggestions](https://github.com/search?q=org%3Aopenfeatureflags+label%3Aroadmap-proposal&type=issues)
+
+## Decision making
+
+Decisions are made by a consensus of [Interested Parties](./interested-parties.md).
```

Comment:
It is not clear how to become an interested party or what the responsibilities and privileges are of an interested party. Currently the interested parties are all listed as companies, not individual contributors. Does this mean each of these companies will always have the same power regardless of if their contributors move away from the project? This model also strongly discourages open source contributors who are not affiliated with a company or with a company that wants to make a public declaration of support for the project. Most CNCF projects I'm aware of don't have such a corporate-first model of governing. I would suggest you define roles such as technical committee, governance committee, and maintainer, and make it very clear what the requirements are to move to the next rung of the ladder and what the expectations/permissions/responsibilities are for each. I would also caution against having "founder" be a position at all because you would hope the project lasts longer than any person's involvement or employment at a particular company.

Reply:
> It is not clear how to become an interested party or what the responsibilities and privileges are of an interested party. 

The was a sentence about submitting a pull request. Created https://github.com/openfeatureflags/governance/pull/10 to make things more explicit

> Currently the interested parties are all listed as companies, not individual contributors. This model also strongly discourages open source contributors....

I do not think so, the first field was for an individual. Anyway https://github.com/openfeatureflags/governance/pull/10 makes it much more explicit

> Most CNCF projects I'm aware of don't have such a corporate-first model of governing

Well... things in CNCF projects are quite complicated

> I would suggest you define roles such as technical committee, governance committee, and maintainer, and make it very clear what the requirements are to move to the next rung of the ladder and what the expectations/permissions/responsibilities are for each.

This is in the scope for the full governance model. This document is only the **bootstrap** governance document. Created https://github.com/openfeatureflags/governance/issues/11 to be explicit about the expectations for the governance document

> I would also caution against having "founder" be a position at all

There might be a Founder position with some additional privileges in the final governance. I would not support a founder as ultimate decision making body (BDFL alike governance models), and I haver a story of fighting against it in Jenkins.

For the bootstrap governance, having an ultimate decision making body is common and somewhat reasonable to speed up the things. I am waiting feedback/decision by @beeme1mr on this matter